src/main/java/ca/bc/gov/backendstartapi/util/Empty.java

```diff
@@ -0,0 +1,16 @@
+package ca.bc.gov.backendstartapi.util;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/** This interface should be used to identify an empty object. */
+public interface Empty {
+
+  /**
+   * Check if a given object is empty. This method should be implemented for every class that
+   * subscribe to it.
```

Comment:
> This method should be implemented for every class that subscribe to it.

I think we should remove this bit, as this is true for any interface method without a standard implementation.

Reply:
Indeed. Removing it.